lnrpc/invoicesrpc/addinvoice_test.go

```diff
@@ -683,6 +689,25 @@ var populateHopHintsTestCases = []struct {
 			},
 		},
 	},
+}, {
+	name: "consider all the open channels up to maxHopHints " +
```

Comment:
strictly speaking this is testing the `consider all the open channels` part, not the `up to maxHopHints` part. I think you'll add a test case using `maxHopHints : 1` to check that.

Reply:
Good point, I'll add that test.